pages/templates/data/_tags.ts

```diff
@@ -0,0 +1,122 @@
+type Tag = { id: Lowercase<string>; displayValue: string };
```

Comment:
these files cannot be inside `pages` as next will try (and fail) to render them as a page which will break the build, instead just make a new folder inside the `/data` folder at the root and then import from there :)